tests/test_images.py

```diff
@@ -255,3 +255,35 @@ def test_cmyk_no_filter():
     name = "iss2522.pdf"
     reader = PdfReader(BytesIO(get_data_from_url(url, name=name)))
     reader.pages[0].images[0].image
+
+
+@pytest.mark.enable_socket()
+def test_separation_1byte_to_rgb_inverted():
+    """Cf #2343"""
+    url = "https://github.com/py-pdf/pypdf/files/13679585/test2_P038-038.pdf"
+    name = "iss2343.pdf"
+    reader = PdfReader(BytesIO(get_data_from_url(url, name=name)))
+    url = (
+        "https://private-user-images.githubusercontent.com/4083478/"
```

Comment:
This should probably use the resources directory - JWTs will expire after some time and might even depend on the logged-in user.